### PR TITLE
Saul 234 - Update arguments help descriptions

### PR DESCRIPTION
**Pull request by damatojuan**

### CODE REVIEW

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py [hunk 1]
```diff
@@ -31,8 +31,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     transform_name_arg_type = CLIArgumentType(options_list=['--transform-name', '-t'], metavar='TRANSFORM_NAME')
     expiry_arg_type = CLIArgumentType(options_list=['--expiry'], type=datetime_format, metavar='EXPIRY_TIME')
     default_policy_name_arg_type = CLIArgumentType(options_list=['--content-key-policy-name'], help='The default content key policy name used by the streaming locator.', metavar='DEFAULT_CONTENT_KEY_POLICY_NAME')
-    correlation_data_type = CLIArgumentType(validator=validate_correlation_data, help="Space-separated list of customer provided correlation data that will be returned in Job completed events in key=value format.", nargs='*', metavar='CORRELATION_DATA')
-    token_claim_type = CLIArgumentType(validator=validate_token_claim, help='Space-separated list of required token claims in key=value format.', nargs='*', metavar='ASYMMETRIC TOKEN CLAIMS')
+    correlation_data_type = CLIArgumentType(validator=validate_correlation_data, help="Space-separated correlation data in 'key[=value]' format. This customer provided data will be returned in Job and JobOutput state events.", nargs='*', metavar='CORRELATION_DATA')
```
Comment (NahuelBerg) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py:34]: I think it should begin with "Space-searated _list_ of" to keep consistency with other parameters.

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py [hunk 2]
```diff
@@ -31,8 +31,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
...
+    token_claim_type = CLIArgumentType(validator=validate_token_claim, help="Space-separated required token claims in '[key=value]' format.", nargs='*', metavar='ASYMMETRIC TOKEN CLAIMS')
```
Comment (NahuelBerg) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py:35]: Same here

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py [hunk 3]
```diff
@@ -237,8 +237,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('ams streaming-endpoint create') as c:
         c.argument('tags', arg_type=tags_type)
         c.argument('description', help='The streaming endpoint description.')
-        c.argument('scale_units', help='The number of scale units.')
-        c.argument('availability_set_name', help='AvailabilitySet name.')
+        c.argument('scale_units', help='The number of scale units.  Use the Scale operation to adjust this value.')
```
Comment (NahuelBerg) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py]: Why 2 spaces after 'units.'?

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py [hunk 4]
```diff
@@ -282,24 +282,24 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
...
         c.argument('encoding_type', arg_type=get_enum_type(LiveEventEncodingType),
-                   arg_group='Encoding', help='The encoding type for live event.')
-        c.argument('preset_name', arg_group='Encoding', help='The encoding preset name.')
+                   arg_group='Encoding', help='The encoding type for live event. This value is specified at creation time and cannot be updated.')
+        c.argument('preset_name', arg_group='Encoding', help='The encoding preset name. This value is specified at creation time and cannot be updated.')
         c.argument('tags', arg_type=tags_type)
         c.argument('key_frame_interval_duration', arg_group='Input',
                    help='ISO 8601 timespan duration of the key frame interval duration.')
-        c.argument('access_token', arg_group='Input', help='The access token.')
+        c.argument('access_token', arg_group='Input', help='A unique identifier for a stream.  This can be specified at creation time but cannot be updated.  If omitted, the service will generate a unique value.')
```
Comment (NahuelBerg) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py]: Same here with the 2 spaces

**Review by NahuelBerg — APPROVED** [review]

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py (hunk 4 above)
Comment (damatojuan) [reply src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py]: copy/paste, will fix

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py (hunk 3 above)
Comment (damatojuan) [reply src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py]: copy/paste, will fix

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py (hunk 1 above)
Comment (damatojuan) [reply src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py:34]: Changed to follow CLI standards

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py (hunk 2 above)
Comment (damatojuan) [reply src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py:35]: Changed to follow CLI standards

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py [hunk 5]
```diff
@@ -282,24 +282,24 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
...
-        c.argument('preview_locator', arg_group='Preview', help='The preview locator Guid.')
-        c.argument('streaming_policy_name', arg_group='Preview', help='The name of streaming policy used for live event preview.')
-        c.argument('alternative_media_id', arg_group='Preview', help='An alternative media identifier associated with the preview URL. This identifier can be used to distinguish the preview of different live events for authorization purposes in the custom license acquisition URL template or the custom key acquisition URL template of the streaming policy specified in the streaming policy name field.')
-        c.argument('vanity_url', action='store_true', help='The live event vanity URL flag.')
+        c.argument('preview_locator', arg_group='Preview', help='The identifier of the preview locator in Guid format. Specifying this at creation time allows the caller to know the preview locator url before the event is created.  If omitted, the service will generate a random identifier.  This value cannot be updated once the live event is created.')
```
Comment (Renkon) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py]: double space?

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py [hunk 6]
```diff
@@ -282,24 +282,24 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
...
         c.argument('encoding_type', arg_type=get_enum_type(LiveEventEncodingType),
-                   arg_group='Encoding', help='The encoding type for live event.')
-        c.argument('preset_name', arg_group='Encoding', help='The encoding preset name.')
+                   arg_group='Encoding', help='The encoding type for live event. This value is specified at creation time and cannot be updated.')
+        c.argument('preset_name', arg_group='Encoding', help='The encoding preset name. This value is specified at creation time and cannot be updated.')
         c.argument('tags', arg_type=tags_type)
         c.argument('key_frame_interval_duration', arg_group='Input',
                    help='ISO 8601 timespan duration of the key frame interval duration.')
-        c.argument('access_token', arg_group='Input', help='The access token.')
+        c.argument('access_token', arg_group='Input', help='A unique identifier for a stream. This can be specified at creation time but cannot be updated.  If omitted, the service will generate a unique value.')
```
Comment (Renkon) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py]: double space?

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py [hunk 7]
```diff
@@ -282,24 +282,24 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
...
-        c.argument('vanity_url', action='store_true', help='The live event vanity URL flag.')
+        c.argument('preview_locator', arg_group='Preview', help='The identifier of the preview locator in Guid format. Specifying this at creation time allows the caller to know the preview locator url before the event is created.  If omitted, the service will generate a random identifier.  This value cannot be updated once the live event is created.')
+        c.argument('streaming_policy_name', arg_group='Preview', help='The name of streaming policy used for the live event preview. This can be specified at creation time but cannot be updated.')
+        c.argument('alternative_media_id', arg_group='Preview', help='An Alternative Media Identifier associated with the StreamingLocator created for the preview.  This value is specified at creation time and cannot be updated.  The identifier can be used in the CustomLicenseAcquisitionUrlTemplate or the CustomKeyAcquisitionUrlTemplate of the StreamingPolicy specified in the StreamingPolicyName field.')
```
Comment (Renkon) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py]: double space?

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py [hunk 8]
```diff
@@ -282,24 +282,24 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
...
+        c.argument('vanity_url', action='store_true', help='Specifies whether to use a vanity url with the Live Event.  This value is specified at creation time and cannot be updated.')
         c.argument('client_access_policy', arg_group='Cross Site Access Policies', help='The local full path to the clientaccesspolicy.xml used by Silverlight.')
         c.argument('cross_domain_policy', arg_group='Cross Site Access Policies', help='The local full path to the crossdomain.xml used by Silverlight.')
-        c.argument('stream_options', nargs='+', arg_type=get_enum_type(StreamOptionsFlag), help='The stream options.')
+        c.argument('stream_options', nargs='+', arg_type=get_enum_type(StreamOptionsFlag), help='The options to use for the LiveEvent.  This value is specified at creation time and cannot be updated.')
```
Comment (Renkon) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py]: double space?

**Review by Renkon — CHANGES_REQUESTED** [review]

**Pull request merged**